internal/core/src/exec/Driver.cpp

```diff
@@ -78,11 +80,21 @@ DriverFactory::CreateDriver(std::unique_ptr<DriverContext> ctx,
                            plannode)) {
             operators.push_back(std::make_unique<PhyVectorSearchNode>(
                 id, ctx.get(), vectorsearchnode));
-        } else if (auto groupbynode =
-                       std::dynamic_pointer_cast<const plan::GroupByNode>(
+        } else if (auto vectorGroupByNode =
+                       std::dynamic_pointer_cast<const plan::SearchGroupByNode>(
+                           plannode)) {
+            operators.push_back(std::make_unique<PhySearchGroupByNode>(
+                id, ctx.get(), vectorGroupByNode));
+        } else if (auto queryGroupByNode =
+                       std::dynamic_pointer_cast<const plan::AggregationNode>(
+                           plannode)) {
+            operators.push_back(std::make_unique<PhyAggregationNode>(
+                id, ctx.get(), queryGroupByNode));
+        } else if (auto projectNode =
+                       std::dynamic_pointer_cast<const plan::ProjectNode>(
                            plannode)) {
             operators.push_back(
-                std::make_unique<PhyGroupByNode>(id, ctx.get(), groupbynode));
+                std::make_unique<PhyProjectNode>(id, ctx.get(), projectNode));
         }
```

Comment:
for query_group_by with filter expr, the pipeline is:
agg_operator--->project_operator--->filterbits_operator-->mvcc_operator
no changes towards existing framework existing operator